symplyphysics/laws/optics/interference_from_two_holes.py

```diff
@@ -0,0 +1,44 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression,
+                           validate_input, validate_output)
+
+# Description
+## If two monochromatic waves of the same frequency are combined in the Young scheme,
+## then for the case when d << L and x << L (L - distance from holes to surface with image,
+## x - coordinate from symmetry point, d - distance), it is possible to approximate:
+
+# Law: delta = x * d / l
+# Where:
+## delta - optical travel difference for two coherent waves
+## x - coordinate in interference picture
+## d - distance between holes (or source parameter)
+## l - distance from holes to picture
+
+# Conditions:
+# - Two monochromatic waves of the same frequency;
+# - d << L and x << L
```

Comment:
What if slits are not symmetrical to the source of light?

Reply:
A fair point. It is worth mentioning this in the conditions. After all, this formula is calculated for systems similar to Young's scheme.